cakechat/dialog_model/inference/candidates/beamsearch.py

```diff
@@ -1,6 +1,7 @@
-from itertools import izip_longest
+from six.moves import zip_longest
```

Comment:
It's better to group all the new six imports together (near the third-party import sections)